site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -2,4 +2,79 @@
 api: runtime
 ---
 
-<!-- Intentionally blank -->
+## Examples
+
+### Use [`getURL`][getURL] to add an extension image to a page {: #example-getURL }
```

Comment:
I recommend adding text between these headings* between these titles, such as:

"The following sections demonstrate a number of ways to use the chrome.runtime API."

* https://developers.google.com/style/headings#formatting-a-heading-or-title